components/style_derive/cg.rs

```diff
@@ -406,3 +406,41 @@ pub fn where_predicate(
         )],
     })
 }
+
+/// Transforms "FooBar" to "foo-bar".
+///
+/// If the first Camel segment is "Moz" or "Webkit", the result string
+/// is prepended with "-".
+pub fn to_css_identifier(mut camel_case: &str) -> String {
+    camel_case = camel_case.trim_right_matches('_');
+    let mut first = true;
+    let mut result = String::with_capacity(camel_case.len());
+    while let Some(segment) = split_camel_segment(&mut camel_case) {
+        if first {
+            match segment {
+                "Moz" | "Webkit" => first = false,
```

Comment:
We should also add `"Servo"` here.

Reply:
This is just moving code around, but yeah, you're right.